src/appkit.cs

```diff
@@ -27059,4 +27059,376 @@ interface NSCollectionLayoutEdgeSpacing : NSCopying, INSCopying
 		[NullAllowed, Export ("bottom")]
 		NSCollectionLayoutSpacing Bottom { get; }
 	}
+
+	[NoiOS, Mac (10,15)]
+	[BaseType (typeof(NSCollectionLayoutItem))]
```

Comment:
minor, style, space missing.
```suggestion
	[BaseType (typeof (NSCollectionLayoutItem))]
```